src/web/reducers/flows.js

```diff
@@ -256,6 +258,40 @@ reducer = reduceReducers(
         }
       }),
 
+      [copyFlowNode]: state => ({
+        ...state,
+        nodeInBuffer: { ..._.find(state.flowsByName[state.currentFlow].nodes, { id: state.currentFlowNode }) }
```

Comment:
Don't forget to set the initial state to let developers know about the structure of the state

Reply:
I will do the modification as we need to merge tonight